CHANGELOG.md

```diff
@@ -123,6 +123,8 @@ New modules
 
 * `Data.Sign.Show` to show a sign
 
+* `AlgebraPropreties.Semigroup.Reasoning` adding reasoning combinators for semigroups
```

Comment:
Lots of typos in the name?

Reply:
And indeed, if we do as I have suggested several times, and *not* make this a new module, but under `additions to existing modules`, then it should be under `Algebra.Properties.Semigroup`...